gulpfile.js

```diff
@@ -106,9 +106,11 @@ var cssFilePaths = [];
 var jsFilePaths = [];
 var fontFolderPaths = [];
 var cssBackgroundFilepaths = [];
+// This cache slug should be same as feconf.CACHE_SLUG_DEV
```

Comment:
Should we keep `feconf.CACHE_SLUG_DEV`, but make it a must that it should be empty?